src/phpFastCache/Drivers/Mongodb/Driver.php

```diff
@@ -59,11 +61,16 @@ public function __construct(array $config = [])
      */
     public function driverCheck()
     {
-        if(class_exists('MongoDB\Driver\Manager')){
-            trigger_error('PhpFastCache currently only support the pecl Mongo extension.<br />
-            The Support for the MongoDB extension will be added coming soon.', E_USER_ERROR);
+        if(!class_exists('MongoDB\Driver\Manager')){
+            trigger_error('This driver is used to support the pecl MongoDb extension with mongo-php-library.<br />
```

Comment:
Requires a Composer require, new MongoDB driver is a low-level driver, you have only database commands, without a high level classes (Collection, Database, etc).